### PR TITLE
CI:DOCS: Document API docs + CORS maintenance

### PR DESCRIPTION
**Pull request by cevich**
Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

File: docs/Readme.md [hunk 1]
```diff
@@ -26,3 +26,14 @@ link on that page.
 | ------------------------------------ | --------------------------- |
 | docs/remote-docs.sh | Read the docs/source/markdown files and format for each platform |
 | docs/links-to-html.lua | pandoc filter to do aliases for html files |
+
+## API Reference
+
+The [lates tonline documentation](http://docs.podman.io/en/latest/_static/api.html) is
```
Comment (baude) [inline docs/Readme.md:32]: typo

File: docs/Readme.md [hunk 2]
```diff
@@ -26,3 +26,14 @@ link on that page.
...
+automatically generated from commited upstream sources.  There is a short-duration
+cache involved, in case old content or an error is returned, try clearing your browser
+cache or returning to the site after 10-30 minutes.
+
+***Maintainers Note***: Please referr to [the Cirrus-CI tasks
```
Comment (TomSweeneyRedHat) [inline docs/Readme.md:37]: referr -> refer

**Comment by TomSweeneyRedHat** [comment]
Other than two typos, looks really good, thx @cevich

**Comment by mheon** [comment]
Reopening since I assume Tom accidentally closed

**Comment by TomSweeneyRedHat** [comment]
@mheon, DOH! yes and ty!

**Comment by cevich** [comment]
Thanks @TomSweeneyRedHat and @baude I fixed those items, rebased, and force-pushed.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**